tools/test_deleted_renamed_referenced_modules

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+FILES="${@}"
+success=1
+for FILE in $FILES; do
+    if [ -f "$FILE" ]; then
+        # if file exists it means it was renamed, and then original file name is retrieved by git log
+        FILE=$(git log --follow -p $FILE | grep 'rename from tests/')
+    fi
+    # module name as appears in scheduling files excluding 'tests/' and file extension
+    module=$(echo $FILE | cut -f 2- -d '/' | cut -f 1 -d '.')
+    target_paths='schedule/ products/*/main.pm lib/main_common.pm'
```

Comment:
In general, I guess it's fine to complain if module is referenced in the comments, but do we want to be that strict?
For `main.pm` part we can use `loadtest` keyword, for instance.

WDYT?

Reply:
I thought about it, I think if I would see a useful comment with wrong file name or deleted in those files I would fix it as well, so it is good that fails there, plus makes the code more simple.

Reply:
Yeah, but comment might be like "reworked module bla/bla1", but error message is clear enough, so let;s merge.